bottlecap/src/config/env.rs

```diff
@@ -230,3 +234,116 @@ where
         _ => Err(serde::de::Error::custom("expected a string or an integer")),
     }
 }
+
+pub(crate) fn deserialize_additional_endpoints<'de, D>(
```

Comment:
Could you move this to its own file? 

I'm trying to clean config in another PR, and this is used both in yaml and env, so might as well separate it